docs/plugins/contributors/gatsby-node.js

```diff
@@ -0,0 +1,78 @@
+const { Octokit } = require("@octokit/rest");
+const path = require("path");
+
+const NODE = `contributor`;
+
+exports.sourceNodes = async ({ actions, createContentDigest, createNodeId }, { repo, owner }) => {
+  const { createNode, createTypes } = actions;
+
+  createTypes(
+    `
+    type Contributor implements Node {
+      id: ID!
+      login: String
+      avatar_url: String
+      url: String
+      name: String
+      blog: String
+      bio: String
+      email: String
+      location: String
+      twitter_username: String
+      company: String
+    }
+  `,
+  );
+
+  try {
+    require("dotenv-safe").config({
+      example: path.resolve(__dirname, `../../../.env.example`),
+      path: path.resolve(__dirname, `../../../.env`),
+      allowEmptyValues: true,
+    });
+
+    const octokit = new Octokit({ auth: process.env.GH_TOKEN });
+    const names = [];
+
+    await octokit.repos
+      .listContributors({
+        owner,
+        repo,
+      })
+      .then(c => names.push(...c.data.map(({ login }) => login)));
+
+    const reqs = names.map(username => octokit.users.getByUsername({ username }));
+    const users = await Promise.all(reqs).then(all => all.map(n => n.data));
+
+    users.forEach(u => {
+      createNode({
+        ...u,
+        id: createNodeId(`${NODE}-${u.id}`),
+        parent: null,
+        children: [],
+        internal: {
+          type: NODE,
+          content: JSON.stringify(u),
+          contentDigest: createContentDigest(u),
+        },
+      });
+    });
+  } catch (error) {
+    if (error.missing.includes("GH_TOKEN")) {
+      console.warn("Missing an access token for GitHub. Please create one");
+      console.info(
+        "Token is needed to display Contributors component, which is based on github api",
+        "How to create github personal token: https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token",
+        "Do not forget to include required token scopes: all repo scopes and read:org",
+        "After you will create token in github, do not forget to create .env file in the root folder with GH_TOKEN=<YOUR TOKEN>",
+      );
+    } else {
+      console.error(error);
+      console.info(
+        "It is possible that you forgot to include repo and read:org scopes in your personal github token",
+      );
```

Comment:
Wow, this is a great addition, thanks for this! 🥇

Reply:
Are these definitely the scopes needed? I thought I gave my token only read:org. Did you test it?